test/lib.ResultSet.js

```diff
@@ -338,6 +338,35 @@ describe('Lib', function () {
       });
 
     it(
+      'should fetch all rows with a LOB and rowsAsArray set',
```

Comment:
Although there are test cases that are _not_ properly formatted, in this file there are still test cases that _are_ properly formatted. Better use proper formatting for your test as well `it('...', function (done) {   .......`.